source/Nevermore/Transient/TransactionExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Data;
+using System.Data.Common;
+using Microsoft.Data.SqlClient;
+
+
+namespace Nevermore.Transient
+{
+    public static class TransactionExtensions
+    {
+        public static DbTransaction BeginTransactionWithRetry(this SqlConnection connection, IsolationLevel isolationLevel, string sqlServerTransactionName)
+        {
+            return BeginTransactionWithRetry(connection, isolationLevel, sqlServerTransactionName, RetryManager.Instance.GetDefaultSqlTransactionRetryPolicy());
+        }
+
+        public static DbTransaction BeginTransactionWithRetry(this SqlConnection connection, IsolationLevel isolationLevel, string sqlServerTransactionName, RetryPolicy retryPolicy)
+        {
+            return (retryPolicy ?? RetryPolicy.NoRetry).LoggingRetries("Beginning Database Transaction").ExecuteAction(() => connection.BeginTransaction());
```

Comment:
```suggestion
            return (retryPolicy ?? RetryPolicy.NoRetry).LoggingRetries("Beginning Database Transaction").ExecuteAction(() => connection.BeginTransaction(isolationLevel, sqlServerTransactionName));
```
Don't we need to continue passing those arguments?

Reply:
🤦  Yes, haha of course. Thanks for catching this :D